README.md

```diff
@@ -188,6 +189,12 @@ To run against a production instance, provide a valid GoogleServices-Info.plist
 [public](https://firebase.google.com/docs/database/security/quickstart) while your tests are
 running.
 
+### Firebase Performance
+If you're doing specific Firebase Performance development, see
+[the Performance README](FirebasePerformance/README.md) for instructions about building the SDK
```

Comment:
We can use the full name `Firebase Performance Monitoring` here and the rest of doc.

Reply:
Fixed.